debian/rules

```diff
@@ -50,37 +50,26 @@ DEPENDS += grub-pc, \
 # our appliance requires a few more packages than would be pulled in by
 # "ubuntu-minimal", and these additional packages are listed here.
 #
-DEPENDS += adduser, \
-	   ansible, \
+DEPENDS += ansible, \
 	   apt, \
-	   apt-utils, \
 	   auditd, \
-	   bzip2, \
 	   cloud-init, \
 	   console-setup, \
 	   cron, \
-	   debconf, \
-	   debconf-i18n, \
 	   debootstrap, \
 	   debsums, \
 	   dmidecode, \
-	   eject, \
 	   init, \
-	   initramfs-tools, \
```

Comment:
could you provide an `aptitude why initramfs-tools`? I'm just curious, since it looks like this is still installed..

and the only reason I'm asking, is I'm curious if we actually do need it, in order to update the initramfs for kernel upgrades.. perhaps the kernel packages depend on it?

Reply:
Looks like ZFS:
```
root@seb-after:~# aptitude why initramfs-tools
i   delphix-platform-esx Depends delphix-zfs
i A delphix-zfs          Depends zfs-initramfs (= 2.1.99-delphix+2023.06.15.09)
i A zfs-initramfs        Depends initramfs-tools
```